lib/spack/spack/cmd/dev_build.py

```diff
@@ -92,6 +102,13 @@ def dev_build(self, args):
     if args.no_checksum:
         spack.config.set('config:checksum', False, scope='command_line')
 
+    abstract_specs = spack.cmd.parse_specs(args.spec)
+    tests = False
+    if args.test == 'all':
+        tests = True
+    elif args.test == 'root':
+        tests = [spec.name for spec in abstract_specs]
```

Comment:
```suggestion
        tests = [package.name]
```
This command can only take one spec at a time, and we've already checked that we only got one. We can just assign it directly.

Reply:
Good point, thanks.